CHANGELOG

```diff
@@ -1,5 +1,10 @@
 CHANGELOG:
 
+2022-03-05 Alexandros K. Antonakakis <antonakakis1@gmail.com>
+   - Added support for python3 in verify-turnin
```

Comment:
I would change this to "Add", both here and below, to match the style of the rest of the entries.